### PR TITLE
Describe that FLR settings do not apply to TimeoutManager in v6

### PR DESCRIPTION
**Pull request by MarcinHoppe**
As of v6, the TimeoutManager has its own fixed FLR policy.

This is doco PR for Particular/NServiceBus#3337.

### CODE REVIEW

**Comment by SimonCropp** [comment]
should this page also be updated http://docs.particular.net/nservicebus/errors/automatic-retries  ?

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -147,6 +147,11 @@ Configuration should be easy to apply to your code and easy to manipulate by han
 `IQueryTimeouts` implements the concern of polling for timeouts outside the context of a message pipeline. `IPersistTimeouts` implements the concern of storage and removal for timeouts which is executed inside the context of a pipeline. Depending on the design of the timeout persisters, those concerns can now be implemented independently. Furthermore, `IPersistTimeouts` introduced a new parameter `TimeoutPersistenceOptions `. This parameter allows access to the pipeline context. This enables timeout persisters to manipulate everything that exists in the context during message pipeline execution.
 
 
+### Automatic retries
+
+Previously configuring the number of times a message will be retried by the First Level Retries (FLR) mechanism also determined how many times the `TimeoutManager` attempted to retry dispatching a defered message in case an exception was thrown. Starting from version 6, the `TimeoutManager` will attempt to attempt the dispatch five times (this number is not configurable anymore). Configuration of the FLR mechanism for non-defered message dispatch has not been changed.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: isnt it deferred  ?

**Comment by MarcinHoppe** [comment]
@SimonCropp Good catch. I updated the FLR doco as well.

File: nservicebus/upgrades/5to6.md (hunk 1 above)
Comment (MarcinHoppe) [inline nservicebus/upgrades/5to6.md]: @SimonCropp Of course it is. Fixed.

File: nservicebus/errors/automatic-retries.md [hunk 2]
```diff
@@ -50,6 +50,9 @@ snippet:FLRConfigurationSource
 snippet:FLRConfigurationSourceUsage
 
 
+NOTE: Starting from version 6, configuration of the FLR mechanism will have no effect on how many times deferred message are dispatched when exception is thrown. In such case the `TimeoutManager` will attempt the dispatch five times.
```
Comment (SimonCropp) [inline nservicebus/errors/automatic-retries.md]: "such cases" or "such as case"?

Comment (MarcinHoppe) [inline nservicebus/errors/automatic-retries.md]: Fixed.

**Pull request merged**